web/control/components/UserReadHistory.js

```diff
@@ -14,9 +13,7 @@ export function UserReadHistory({
   return (
     <History
       key={1}
-      missions={missions.filter(
-        m => m.startTime >= getStartOfMonth(now() - 183 * DAY)
-      )}
```

Comment:
This (useless?) filter was removed, history for not logged controlers is now limited to 2 months as the default history.